documentation/asciidoc/stories/assembly_using_spring_session.adoc

```diff
@@ -0,0 +1,9 @@
+[id='sb_starter_session']
```

Comment:
This is duplicated. 
The `documentation/asciidoc/stories/assembly_running_server.adoc` already includes the `Using Spring Session` topic
```
include::{topics}/proc_enabling_spring_session.adoc[leveloffset=+1]
```
I guess it makes sense to keep this one and remove from the "server" topic?

Reply:
cc @karesti WDYT?

Reply:
great catch, Pedro. I've remove the one from the "server" topic.